api/templates/design.go

```diff
@@ -40,13 +44,21 @@ var index = ResultType("application/vnd.templates.index", func() {
 	})
 })
 
+var tokenSecurity = APIKeySecurity("storage-api-token", func() {
+	Description("Storage Api Token Authentication")
+})
+
 var _ = Service("templates", func() {
 	Description("Service for applying templates to Keboola projects")
-	cors.Origin("*", func() {
-		cors.Headers("X-StorageApi-Token")
-	})
+	// CORS
+	cors.Origin("*", func() { cors.Headers("X-StorageApi-Token") })
+
+	// Set authentication by token to all endpoints without NoSecurity()
+	Security(tokenSecurity)
+	defer AddTokenHeaderToPayloads(tokenSecurity, "storageApiToken", "X-StorageApi-Token")
```

Comment:
Goa vyzaduje definovat rucne, ako sa spracuje token v danom endpoint/payload.
Moze byt ulozeny v URL, header, post ....

My chceme token vzdy v `X-StorageApi-Token` hlavicke, ... preto som napisal tuto funkciu, ktora to zabezpeci.

Ak je niekde `NoSecurity()`, tak tam token nebude.